### PR TITLE
updated routes module to latest generator

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | -
| Refs tickets      | -
| License           | MIT
| Changelog updated | no

### CODE REVIEW

File: src/system/RoutesModule/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+deploy:
+  provider: releases
+  api_key:
+    secure: "" # Enter your api key here!
+  file: ZikulaRoutesModule.zip
+  on:
+    tags: true
+    repo: zikula/RoutesModule
+
```
Comment (cmfcmf) [inline src/system/RoutesModule/.travis.yml]: This file doesn't really make sense.

File: src/system/RoutesModule/Base/RoutesModuleInstaller.php [hunk 2]
```diff
@@ -82,206 +79,205 @@ public function upgrade($oldVersion)
...
+    
     /**
      * Renames this application in the core's extensions table.
      */
     protected function updateExtensionInfoFor140()
     {
         $conn = $this->getConnection();
         $dbName = $this->getDbName();
-
+
```
Comment (cmfcmf) [inline src/system/RoutesModule/Base/RoutesModuleInstaller.php]: Lot's of added whitespace...

Comment (Guite) [inline src/system/RoutesModule/Base/RoutesModuleInstaller.php]: This is something I can not fix, caused by https://bugs.eclipse.org/bugs/show_bug.cgi?id=398172

File: src/system/RoutesModule/.travis.yml (hunk 1 above)
Comment (Guite) [inline src/system/RoutesModule/.travis.yml]: @cmfcmf https://github.com/Guite/core/compare/a95b035a0c5537e136c76c12e03c921b79c3944f...45e17701664cce96ce769b566ca3315c6003f888

**Pull request merged**